docs/prompts-2026-08-08.md

```
@@ -0,0 +1,801 @@
+At review time: **85 local branches, 219 remote branches.** Of those, **85 remote branches are
+already fully merged into `guarzo/zoo`**, as are 32 local ones. The backup branches
+(`backup/guarzo-zoo-prerebase`, `zoo-backup-prerebase`, `backup-pre-rebase-102`) are **not**
+being kept deliberately — they simply have not been cleaned up yet, and can go.
+
+### The task
+
+Deleting branches is destructive and easy to get wrong, so work in this order and **show the
+list before deleting anything**.
+
+1. Refresh, then build the candidate lists:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)